sdk/monitor/opentelemetry-exporter-azuremonitor/src/main/java/com/azure/opentelemetry/exporter/azuremonitor/AzureMonitorExporterBuilder.java

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.opentelemetry.exporter.azuremonitor;
+
+import com.azure.opentelemetry.exporter.azuremonitor.implementation.ApplicationInsightsClientImpl;
+import com.azure.opentelemetry.exporter.azuremonitor.implementation.ApplicationInsightsClientImplBuilder;
+import com.azure.opentelemetry.exporter.azuremonitor.implementation.NdJsonSerializer;
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+import com.azure.core.http.policy.RetryPolicy;
+import com.azure.core.util.Configuration;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.core.util.serializer.JacksonAdapter;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Objects;
+
+/**
+ * This class provides a fluent builder API to instantiate {@link AzureMonitorExporter} that implements
+ * {@link SpanExporter} interface defined by OpenTelemetry API specification.
+ */
+public final class AzureMonitorExporterBuilder {
```

Comment:
ServiceClient Annotations?

Reply:
This doesn't build conventional track 2 clients. It builds an open telemetry exporter. So, I don't think the annotations are appropriate here.